moler/util/moler_test.py

```diff
@@ -30,34 +31,55 @@ def steps_end():
         MolerTest._was_steps_end = True
 
     @staticmethod
-    def error(msg, raise_exception=False):
+    def error(msg, dump=None, raise_exception=False):
         """
         Makes an error (fail the test) and (optional) continue the test flow.
         :param msg: Message to show.
         :param raise_exception: If True then raise an exception, if False then only show msg and mark error in logs.
+        :param dump: If defined then dump object.
         :return: Nothing.
         """
         MolerTest._list_of_errors.append(msg)
-        MolerTest._error(msg, raise_exception)
+        MolerTest._error(msg, raise_exception, dump)
 
     @staticmethod
-    def info(msg):
+    def info(msg, dump=None):
         """
         Shows the message
         :param msg: Message to show.
+        :param dump: If defined then dump object.
         :return: Nothing.
         """
+        if dump is not None:
```

Comment:
This if statement is 3 times repeated in 3 methods. Maybe move it to _dump method?

Reply:
I moved it to **_get_string_message** method